UI/PongWars/src/global.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
@Kunal22shah fyi there are some CI Build errors if you can take a look at it please